### ISSUE TITLE
Ability to run multiple test suites em parallel

### ISSUE THREAD

**Issue by marceloleitner** [sctp/sctp-tests#14]
Running multiple test suites allows us to stress the stack in different ways, hopefully revealing more bugs.
What is mainly needed is another prefix to the netns and log files.

**Comment by marceloleitner** [comment]
Cc @lxin

**Comment by lxin** [comment]
I guess this means one test case can be running in different threads at the same time, and right, some of the test cases need to isolate the logs.
My question is: do you hope ./sctp-tests do this by itself, or users start more ./sctp-tests thread with different param to run the same case?

**Comment by marceloleitner** [comment]
Although this would be nice to have, this is very hard to support properly because we would have to handle situations like possibly building the same test app from src multiple times in parallel.
I'm now thinking that instead of doing this and #13 we could add some 'background noise generator', like a couple of sctp_test that gets started while preparing the topology for the test case, and gets killed when the test case is done. This would be easier and would achieve the initial goal.
